src/AttributeReader.php

```diff
@@ -97,9 +100,9 @@ public function getMethodAnnotation(ReflectionMethod $method, $annotationName):
     /**
      * {@inheritDoc}
      */
-    public function getPropertyAnnotations(ReflectionProperty $property): array
+    public function getPropertyAnnotations(ReflectionProperty $reflectionProperty): array
     {
-        $attributesRefs = $property->getAttributes();
+        $attributesRefs = $reflectionProperty->getAttributes();
         $attributes = [];
```

Comment:
I've added more prectise naming, just to avoid guessing what "property" is (annotation, attribute, reflection, node...)